unbound_exporter.go

```diff
@@ -218,6 +218,18 @@ var (
 			prometheus.CounterValue,
 			nil,
 			"^unwanted\\.replies$"),
+		newUnboundMetric(
+			"recursion_time_avg",
```

Comment:
The Prometheus naming guidelines suggest that the unit of what's encoded should be included in the name, e.g. `recursion_time_seconds_avg`.

Reply:
that's true! And better!